File: common/org.daum.common.genmodel/org.daum.common.genmodel.root/src/main/scala/Materiel.scala
package org.sitac;
/**
 * Created with IntelliJ IDEA.
 * User: jed
 * Date: 05/07/12
 * Time: 14:56
 * To change this template use File | Settings | File Templates.
 */


import org.daum.library.ormH.annotations.Generated
import org.daum.library.ormH.annotations.Id
import org.daum.library.ormH.persistence.GeneratedType

trait Materiel extends org.sitac.SitacContainer {
  @Id
  @Generated(strategy = GeneratedType.UUID)
  var id : java.lang.String = ""

  def getId() : java.lang.String = {
    id
  }

  private var posRef : Option[org.sitac.Position] = null

  private var posTarget : Option[org.sitac.Position] = null

  def getPosRef : Option[org.sitac.Position] = {
    posRef
  }

  def getPosTarget : Option[org.sitac.Position] = {
    posTarget
  }



}

File: javase/org.daum.library.javase.webportal/src/main/java/org/daum/javase/webportal/shared/Materiel.java
package org.daum.javase.webportal.shared;

import com.google.gwt.user.client.rpc.IsSerializable;

/**
 * Created with IntelliJ IDEA.
 * User: max
 * Date: 31/07/12
 * Time: 12:02
 * To change this template use File | Settings | File Templates.
 */
public class Materiel implements IsSerializable {

    private String id;

    public Materiel(){

    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }


}
